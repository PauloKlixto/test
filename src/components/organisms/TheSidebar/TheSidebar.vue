<template>
  <section class="TheSidebar"></section>
</template>

<script>
export default {
  name: 'TheSidebar'
}
</script>

<style lang="scss" scoped>
.TheSidebar {
  background-color: $color-white;
  height: 400px;
}
</style>
