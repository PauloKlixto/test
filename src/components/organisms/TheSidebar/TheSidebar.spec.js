/* eslint-disable no-undef */
import { shallowMount } from '@vue/test-utils'
import TheSidebar from './TheSidebar'

describe('TheSidebar', () => {
  const WRAPPER = shallowMount(TheSidebar)

  test('Componente existe', () => {
    expect(WRAPPER.exists()).toBe(true)
  })

  test('Componente nÃ£o alterado', () => {
    expect(WRAPPER.html()).toMatchSnapshot()
  })
})
