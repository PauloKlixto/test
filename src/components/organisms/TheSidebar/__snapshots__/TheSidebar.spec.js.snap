// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheSidebar Componente n√£o alterado 1`] = `"<section class=\\"TheSidebar\\"></section>"`;
